ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -7,14 +7,25 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.13
+  buildroot:
+    cluster: https://api.ci.openshift.org
+    name: cincinnati-build-root
+    namespace: cincinnati
```

Comment:
I don't understand how the promotion works so I'm asking rather than suggesting: is this namespace correct? I expected to read _cincinnati-ci_ here.